test/CMakeLists.txt

```diff
@@ -276,5 +282,12 @@ endfunction()
 add_subdirectory(unit)
 add_subdirectory(integration)
 add_subdirectory(functional)
-add_subdirectory(package)
-add_subdirectory(example)
+
+# TODO: symlinks used! CK
+# package/pkg-config/src -> ../src/
+# package/FetchContent/src -> ../src/
+# package/find_package/src -> ../src/
+if (UNIX AND NOT PROJECT_IS_TOP_LEVEL)
```

Comment:
(For some reason my comment got lost)

Please don't disable these. Can you explain the issue with git and symlink here? The other runs executed this fine

Reply:
out/install/x64-debug/

Reply:
> out/install/x64-debug/

Need more context, what's the `tree` around it.